Frameworks/Foundation/NSScanner.mm

```diff
@@ -185,45 +150,43 @@ - (BOOL)scanInteger:(int*)valuep {
         }
     }
 
-    return TRUE;
+    return YES;
 }
 
 /**
  @Status Interoperable
 */
 - (BOOL)scanLongLong:(long long*)valuep {
```

Comment:
was not sure how we felt about swapping out large chunks of the previous code. there are also unichar -> char conversion implications I haven't thought about yet, but it should be fine I think...